freepbx/initdb.d/initdb.php

```diff
@@ -111,6 +111,25 @@
 	$sql = 'UPDATE `asterisk`.`freepbx_settings` SET `value` = ? WHERE `keyword` = "HTTPBINDPORT"';
 	$stmt = $db->prepare($sql);
 	$stmt->execute([$_ENV['ASTERISK_WS_PORT']]);
+	$sql = 'UPDATE `asterisk`.`freepbx_settings` SET `value` = ? WHERE `keyword` = "HTTPENABLED"';
+	$stmt = $db->prepare($sql);
+	$stmt->execute([1]);
+}
+
+// Set port for Asterisk WSS from environment
+if (!empty($_ENV['ASTERISK_WSS_PORT'])) {
+	$sql = 'UPDATE `asterisk`.`freepbx_settings` SET `value` = ? WHERE `keyword` = "HTTPTLSBINDPORT"';
+	$stmt = $db->prepare($sql);
+	$stmt->execute([$_ENV['ASTERISK_WSS_PORT']]);
+	$sql = 'UPDATE `asterisk`.`freepbx_settings` SET `value` = ? WHERE `keyword` = "HTTPTLSENABLE"';
```

Comment:
What is the correct syntax?

```suggestion
	$sql = 'UPDATE `asterisk`.`freepbx_settings` SET `value` = ? WHERE `keyword` = "HTTPTLSENABLED"';
```